src/mam4xx/mo_setsox.hpp

```diff
@@ -75,7 +75,7 @@ struct Config {
   int itermax = 20;
   // Real small_value_20 = 1.0e-20;
   // Real small_value_30 = 1.0e-30;
-  int numptrcw_amode[AeroConfig::num_modes()] = {23, 28, 36, 40};
+  int numptrcw_amode[AeroConfig::num_modes()] = {22, 27, 35, 39};
```

Comment:
Is this change related to my species order change? To me, it looks like a fix to accommodate C++'s zero-based array indexing.

Reply:
Yes, it has nothing to do with your species order change and is just a Fortran to C++ indexing problem. But the commit integration tests failed due to the memory error and i needed to fix it to pass all of the tests.